src/blas/backends/netlib/netlib_extensions.cxx

```diff
@@ -537,51 +537,4 @@ sycl::event omatadd(sycl::queue &queue, transpose transa, transpose transb, int6
 #endif
 }
 
-sycl::event omatcopy_batch(sycl::queue& queue, transpose* trans, int64_t* m, int64_t* n,
```

Comment:
These definitions should probably stay here and the definitions starting at line 250 should be modified to buffer (they are currently USM). I'll work on making those changes.

Reply:
I was confused (didn't remember about `netlib_batch.cxx`), I now think both these definitions you already removed and also the ones starting at line 250 should be removed. I think it's easier for you to do that as part of this PR but I can also do so if you want.

Reply:
Thank you! Added these changes to PR.